.test-infra/jenkins/job_sonarqube_report.groovy

```diff
@@ -0,0 +1,53 @@
+/* * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+
+job('beam_sonarqube_report') {
+  commonJobProperties.setTopLevelMainJobProperties(
```

Comment:
Is this post-commit or pre-commit? With --no-parallel I think it should be a once in a while postcommit. But I don't know how to read the params to this method and I'm just doing a quick drive by review. The job & idea seem good to me, but please do make sure a slow job is postcommit.

Reply:
This is a post-commit that runs periodically on master HEAD.

Reply:
As it is post commit and slow anyway, 
would it make sense here to also add some runner tests here? 
test task itself does exclude those
Can not remember state of discussion here... But of course, we might do that later on